src/java/htsjdk/tribble/dbsnp/OldDbSNPCodec.java

```diff
@@ -89,6 +89,11 @@ public OldDbSNPFeature decode(String line) {
     }
 
     @Override
+    public boolean canDecode(String path){
+	return path.toLowerCase().endsWith(".dbsnp") || path.toLowerCase().endsWith(".rod") || path.toLowerCase().endsWith(".snp");
```

Comment:
Um, not sure if this is relevant or not but we just recently deleted OldDbSNPFeature entirely (on the premise that no one uses it anymore).